### DISCUSSION TITLE
sa-update not working

### DISCUSSION THREAD

**Discussion by 132ikl** [docker-mailserver/docker-mailserver#2651] [Q&A]
Hi all, not sure if this is a bug or not so thought I would make a question and promote it to a bug if needed. My SpamAssassin doesn't seem to be updating its mirrorlist properly. Every day I receive this email:

```
/etc/cron.daily/spamassassin:
error: no mirror data available for channel updates.spamassassin.org
channel 'updates.spamassassin.org': MIRRORED.BY file contents were missing, channel failed
sa-update failed for unknown reasons
```

Interestingly, it looks like this file isn't present anywhere on updates.spamassassin.org, but I was able to find it on the [Apache website](https://spamassassin.apache.org/updates/MIRRORED.BY). Here's relevant parts of my mailserver config:

```
$ cat mailserver.env | grep -i spam
# ––– Spamassassin Section ––––––––––––––––––––––
ENABLE_SPAMASSASSIN=1
# deliver spam messages in the inbox (eventually tagged using SA_SPAM_SUBJECT)
SPAMASSASSIN_SPAM_TO_INBOX=1
# spam messages will be moved in the Junk folder (SPAMASSASSIN_SPAM_TO_INBOX=1 required)
MOVE_SPAM_TO_JUNK=0
# add spam info headers if at, or above that level:
# add 'spam detected' headers at that level
# triggers spam evasive actions
# add tag to subject if spam detected
SA_SPAM_SUBJECT=***SPAM*****
```

**Comment by casperklein** [comment]
Works fine for me. Could be a temporary issue?

**Reply by 132ikl** [reply]
Still an issue as of <12 hours ago and I've been getting these emails since I enabled SpamAssassin like 4 days ago. Looking a bit closer into it, I went to go see if `updates.spamassassin.org` was accessible from the container and it seems like all the SSL certificates aren't working? 

```
root@mail:/# update-ca-certificates
Updating certificates in /etc/ssl/certs...
0 added, 0 removed; done.
Running hooks in /etc/ca-certificates/update.d...
done.
root@mail:/# curl https://google.com
curl: (60) SSL: no alternative certificate subject name matches target host name 'google.com'
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

**Reply by casperklein** [reply]
Ok. Please open an issue and provide the details of your setup.

**Reply by 132ikl** [reply]
Done, see #2653
